packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -85,7 +87,9 @@ class AppBar extends StatelessWidget {
   /// last action.
   final List<Widget> actions;
 
-  /// A widget to aid in building scroll-based collapsing app bar effects.
+  /// This widget is stacked behind the toolbar and the tabbar and it is not
```

Comment:
https://www.google.com/design/spec/components/tabs.html#tabs-usage seems to use "tab bar" (and "app bar"). I think only "toolbar" is one word. ("status bar" is certainly two.)

cc @qchong for advice